notebooks/marimo/Exhibit_2_Variance_Plot.py

```
@@ -0,0 +1,242 @@
+    SR0_list,
+    generate_autocorrelated_non_gaussian_data,
+    np,
+    pd,
+    ray,
+    scipy,
+    sharpe_ratio_variance,
+    tqdm,
+):
+    @ray.remote
+    def f3(SR, rho, T, name):
+        X = generate_autocorrelated_non_gaussian_data(T, 10000, SR0=_SR, name=name, rho=rho)
```

Ruff (0.15.10)

[error] Undefined name `_SR`

(F821)